projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.ts

```diff
@@ -14,17 +14,17 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { BsModalService } from 'ngx-bootstrap/modal';
 import { RecordUiService } from '@rero/ng-core';
-import { forkJoin } from 'rxjs';
-import { first } from 'rxjs/operators';
+import { BsModalService } from 'ngx-bootstrap/modal';
 import { ItemsService } from 'projects/admin/src/app/service/items.service';
 import { RecordPermissionService } from 'projects/admin/src/app/service/record-permission.service';
 import { UserService } from 'projects/admin/src/app/service/user.service';
+import { forkJoin } from 'rxjs';
+import { first } from 'rxjs/operators';
```

Comment:
These imports are not used next in the file.

Reply:
This is used on line 99